fork.yaml

```diff
@@ -0,0 +1,107 @@
+title: "layr-labs/nitro-contracts"  # Define the HTML page title
+logo: "logo.png"
+footer: |  # define the footer with markdown
+  [Nitro](https://github.com/Layr-Labs/nitro) fork overview &middot created with [Forkdiff](https://github.com/protolambda/forkdiff)
+base:
+  name: OffchainLabs/nitro-contracts
+  url: https://github.com/OffchainLabs/nitro-contracts
+  hash:  c60b18dc7cf7d1809616d737e9061f3ecc9f28a6
+fork:
+  name: Layr-Labs/nitro-contracts
+  url: https://github.com/Layr-Labs/nitro-contracts
+  ref: refs/heads/eigenda
+def:
+    title: "EigenDA x Arbitrum Nitro Contracts fork diff"
+    description: | # description in markdown
+      The original nitro codebase can be found at [`github.com/OffchainLabs/nitro-contracts`](https://github.com/OffchainLabs/nitro).
+      And the fork at [`github.com/Layr-Labs/nitro-contracts`](https://github.com/Layr-Labs/nitro).
+  
+    sub:
+      - title: "Sequencer Inbox"
+        description: | 
+          Added an `addSequencerL2BatchFromEigenDA` function which process an EigenDA certificate within the inbox. 
+          Certificates are verified within the inbox by calling `verifyBlob` against the `EigenDARollupManager` contract 
+          to remove a trust assumption on the Arbitrum sequencer.
+        globs:
+          - "src/bridge/SequencerInbox.sol"
+          - "src/bridge/ISequencerInbox.sol"
+          - "src/bridge/IBridge.sol"
+          - "src/libraries/Error.sol"
+          - "test/foundry/blob_info/blob_info.json"
+          - "test/storage/SequencerInbox"
+          - "test/foundry/SequencerInbox.t.sol"
+          - "src/mocks/SequencerInboxStub.sol"
+
+      - title:  Blob Verification
+        description: |
+          Introduce two new dependency contracts:
+
+          * `EigenDABlobVerifierL1`: verifies blobs when the rollup parent chain is Ethereum. 
+
+          * `EigenDABlobVerifierL2`: "verifies" blobs when the rollup parent chain isn't Ethereum. This is effectively
+          a noop since EigenDA blobs can only be verified on Ethereum given required access to the `EigenDAServiceManager`.
+
+          Both of this contracts implement the `IRollupManager` interface which is referenced via the `SequencerInbox`.
```

Comment:
"Both of this" -> "Both of *these*"

Reply:
[2d95b36](https://github.com/Layr-Labs/nitro-contracts/pull/40/commits/2d95b369e91021820881550e079be9ccab5ee7a1)